modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated-comparison/segregated-linear-physics.i

```diff
@@ -0,0 +1,80 @@
+mu = 2.6
+rho = 1.0
+advected_interp_method = 'average'
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '0.3'
+    dy = '0.3'
+    ix = '3'
+    iy = '3'
+  []
+[]
+
+[Problem]
+  #TODO: remove both
```

Comment:
Yeah would be cool if we could do this under the hood. I think the previous solution flag is already in your code.

Reply:
on it

Reply:
actually this isnt easy. I dont think the problem should be created by the Physics, and this line creates the sytems. we would need to add APIs to make the problem create new systems from the Physics